src/Model.elm

```diff
@@ -253,6 +257,20 @@ update msg model =
                 Err error ->
                     { model | userTimeline = [], errors = (errorText error) :: model.errors } ! []
 
+        OnLoadUserAccount accountId ->
+            {-
+               @TODO
+               When requesting a user profile, we should load a new "page"
```

Comment:
What do you mean by new page? If you mean updating the URL in the browser, the `Navigation` module exposes `newUrl` and `replaceUrl` which may do what you're after here. But handling browser history (back and fwd) might be more complicated, so you'll probably want to tackle this later.